restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/changelog_depreciated/mariadb/update_script_Aug07_2017.sql

```diff
@@ -0,0 +1,48 @@
+#SQL Script for MySQL/MariaDB to update DB with the schema changes
```

Comment:
@agafox The DB update script needs to be moved to the private repository together with the other DB update scripts